.spelling

```diff
@@ -0,0 +1,49 @@
+APRILTAG
+BODMAS
+camcon
```

Comment:
This is a shortening of "Cambridge Connectors" and may even be a contraction which is unique to SR. Maaaybe stylise as "CamCon"?

Reply:
👍 [28792c8](https://github.com/srobo/docs/pull/320/commits/28792c8b10a1ab0c78a4bb044a6f63bfa085eddf)